New version: QuanEstimationBase v0.1.1

JuliaRegistrator opened the pull request:
- Registering package: QuanEstimationBase
- Repository: https://github.com/QuanEstimation/QuanEstimation.jl
- Created by: @LiuJPhys
- Version: v0.1.1
- Commit: 05cfede3b38bec4ec9e15419f024caf326f5a177
- Reviewed by: @LiuJPhys
- Reference: https://github.com/QuanEstimation/QuanEstimation.jl/commit/05cfede3b38bec4ec9e15419f024caf326f5a177#commitcomment-151881190
- Description: QuanEstimation.jl is an open-source toolkit for quantum parameter estimation. 
<!-- 685e52e240a6a2f73af8e63203aa4b7cfb103a1c794b3e909d97fd070d692aa4b7a9eafc85ad4b7067bf9d175f7b00d5921e87da4e25c42794b90ed608f51506d15087d073bfc88157b8050dfeafe1776ed4f3282b022b3eb37ac2020e2f022be95b5ebb96e967ad50c6914c70b549ddff77181cfe15a99e249a836894cc9f224abb70a3f38dd3556e33db0244ab4719111036e3ee0210c06548920932a93afe05eb855b5434aed1e1f032cebd87e60a2476ba1abfad9b94c053d9576483a99c6415922c56c31217e99aae64e43569d1675581d6a86debbdf0a00288b2a492545454c66f02f5d853160e5b58c79bd036c090ce0a160f71891d9fd634ef1f7edb -->

hmyuuu commented:
[noblock]
We recently reorganized some of our subpackages, moving them from `src/` to `lib/`.  As a result, the [Package.toml](https://github.com/JuliaRegistries/General/pull/123841/files#diff-f5021bc2eb89ba1e17fd18e81e585ad7b76bd7148f062907162b642bc6f5afa0) is changed in this PR and fails the automerge.
Would it be possible to merge this manually?

JuliaRegistrator committed:
New version: QuanEstimationBase v0.1.1

UUID: 9769ca81-ed10-4016-bab9-e66dc61d4d60
Repo: https://github.com/QuanEstimation/QuanEstimation.jl.git
Tree: 91a7171f62ba16ba7193c447bd67768339b1ca35

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
